bin/ci-publish.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+npmPublish() {
+  npm publish
+}
+
+currentVersion() {
+  grep version <package.json | sed 's/[^0-9.]*\([0-9.]*\).*/\1/'
+}
+
+currentProject() {
+  npm ls --depth=-1 | head -1 | cut -f 1 -d " " | sed 's/@.*//'
+}
+
+checkVersion() {
+  local gitTag
+  gitTag=$(git describe --tags)
+
+  if [ "$(currentVersion)" != "$gitTag" ]; then
```

Comment:
> The command finds the most recent tag that is reachable from a commit. If the tag points to the commit, then only the tag is shown. Otherwise, it suffixes the tag name with the number of additional commits on top of the tagged object and the abbreviated object name of the most recent commit.

https://git-scm.com/docs/git-describe